lib/experimental/block-editor-settings-mobile.php

```diff
@@ -23,6 +23,10 @@ function keep_supported_block_editor_settings_mobile( $initial_array, $allow_lis
 		$result = array();
 
 		foreach ( $allow_list_array as $key => $value ) {
+			if ( ! array_key_exists( $key, $initial_array ) ) {
+				continue;
+			}
+
```

Comment:
Thank you for adding this check!  Originally, I had the foreach traversing through the response instead of the allow-list, and when I switched it to the current approach this was in the back of my mind. I just forgot to add this logic before drafting the PR 🤦‍♂️.